### DISCUSSION TITLE
When I use Android to remotely turn on the screen

### DISCUSSION THREAD

**Discussion by rogercc116** [rustdesk/rustdesk#4452] [Feature Request]
### Bug Description

When I use the software to enter the controlled Android device and perform screen rest operation through physical buttons, the screen will go blank 。
But I found a button to light up the screen using similar software, which can trigger the screen to light up. Can we achieve a similar operation？

### How to Reproduce

Use software for remote control after the controlled end is on the screen

### Expected Behavior

Successfully established remote link, but the screen is black

### Operating system(s) on local side and remote side

Android->Android

### RustDesk Version(s) on local side and remote side

1.2.0

### Screenshots

![922734c0a6fc54d6dd800f9e807d990](https://github.com/rustdesk/rustdesk/assets/63380214/ec6624fa-8c9a-4690-94e2-e623bf085151)

**Comment by bigretromike** [comment]
I can confirm, on turn off screen Rustdesk hangs on connection successful and stay there.
